src/sentry/integrations/slack/unfurl/__init__.py

```diff
@@ -0,0 +1,62 @@
+import enum
```

Comment:
@EvanPurkhiser do we ever put code in the `__init__.py` files?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/pipeline/__init__.py I mean I feel like we do?

Maybe @wedamija has a better opinion